pkg/session_recording/recordings.jsx

```diff
@@ -450,6 +450,47 @@
     let RecordingList = class extends React.Component {
         constructor(props) {
             super(props);
+            this.handleColumnClick = this.handleColumnClick.bind(this);
+            this.getSortedList = this.getSortedList.bind(this);
+            this.drawSortDir = this.drawSortDir.bind(this);
+            this.state = {
+                sorting_field: "start",
+                sorting_asc: false,
```

Comment:
This is opposite to the order `this.props.list` is in, did you intend it to be so?